src/cljc/harja/ui/kartta/asioiden_ulkoasu.cljc

```diff
@@ -484,6 +484,12 @@ tr-ikoni {:img (pinni-ikoni "musta")
    {:color puhtaat/vihrea
     :width 6}])
 
+(def talvihoitoreitit
```

Comment:
Nämä selitteet ei nyt täsmää mitä kartalla ja UI:lla näkyy. Tarviiko näitä näkyä? Designerille tästä kysymystä, koska näitä ei Figmassakaan ole. Mutta ei haluta väärääkään tietoa tuohon laittaa

Reply:
Hyvä pointti. Mulla on jäänyt tuo näimmä ihan kesken. Tarkoitus oli tietenkin näyttää noi samalla värillä, kuin mitä ne reititkin. Ja heitin designerillekin ideaa, että esim Traktorireitit tulisi eri värillä kuin Kuorma-autoreitit, mutta se ei vielä ottanut tuulta alleen.

Eli otin pois, kun ei niitä ole speksissäkään.